decentraland/unity-explorer
# fix: zoom out when equipping emote

## What does this PR change?

We have added the expected functionality to the Emote section, where the avatar is always zoomed out and player cannot pan or zoom in, they can only rotate.

## How to test the changes?

Go to the backpack, zoom in, and then change to the emotes section, you should be zoomed out automatically and be unable to zoom in or pan.
